apps/mobile/src/libs/components/gradient-text/gradient-text.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import { Text, View } from "~/libs/components/components";
+import { LinearGradient } from "~/libs/components/linear-gradient";
+import { MaskedView } from "~/libs/components/masked-view";
+import { globalStyles } from "~/libs/styles/styles";
+import {
+	type LinearGradientProps,
+	type StyleProp,
+	type TextStyle,
+} from "~/libs/types/types";
+
+import { styles } from "./styles";
+
+type Properties = {
+	children: string;
+	gradientProps: LinearGradientProps;
+	textStyle: StyleProp<TextStyle>;
```

Comment:
In this case you will have access not only to styles, but to all props like `size` and `preset`

```suggestion
	textProps: ComponentProps<typeof Text>;
```